express-post-json/index.js

```diff
@@ -0,0 +1,27 @@
+const express = require('express');
+
+const app = express();
+
+const grades = {
+
+};
+
+app.get('/api/grades', (req, res) => {
+  const arr = [];
+  for (var key in grades) {
+    arr.push(grades[key]);
+  }
+  res.json(arr);
+});
+
+app.listen(3000, () => {
+  // eslint-disable-next-line no-console
+  console.log('Express server listening on port 3000');
+});
```

Comment:
`app.listen` will work here, but it should be the last thing in your server file.